Jint/Pooling/StringBuilderPool.cs

```diff
@@ -9,7 +9,7 @@ namespace Jint.Pooling
     /// <summary>
     /// Pooling of StringBuilder instances.
     /// </summary>
-    internal sealed class StringBuilderPool
+    internal static class StringBuilderPool
```

Comment:
Why not static anymore?

Reply:
Actually now static instead of sealed, all methods are static and no instances of pool are required 😉